### DISCUSSION TITLE
Using # in Part Names

### DISCUSSION THREAD

**Discussion by Fluxus00** [inventree/InvenTree#3774] [Q&A] [answered]
Hello Guys, 

i just have a short question and i have not seen a similar discussion so far. 

Is it possible to use "#" in a parts name? I am curious, because i am currently trying to import my old partkeepr database on inventree but it seems like Inventree is not allowing a # in a part name.

Now I wonder if there might be some settings i missed or another possibility to use #.

Thank you very much in advance and have a nice day

**Comment by SchrodingersGat** [comment, marked as answer]
Currently `#` characters are disallowed for part names, but this is mostly due to legacy reasons. If you would like to be able to support this, please raise an issue and we can address it.

**Reply by Fluxus00** [reply]
Alright thank you I will do that!
